lib/matplotlib/image.py

```diff
@@ -1167,7 +1167,6 @@ def make_image(self, renderer, magnification=1.0, unsampled=False):
         l, b, r, t = self.axes.bbox.extents
         width = (round(r) + 0.5) - (round(l) - 0.5)
         height = (round(t) + 0.5) - (round(b) - 0.5)
-        # The extra cast-to-int is only needed for python2
         width = int(round(width * magnification))
         height = int(round(height * magnification))
```

Comment:
Should this PR also remove the cast to int indicated by the comment as only beeing needed in py2?

Reply:
The comment was wrong, we need to cast to int in both cases as round returns a float and down-stream code requires an int.